features/users.feature

```diff
@@ -0,0 +1,22 @@
+@users
+Feature: OpenStack Identity
+  As an administrator
+  I want to make sure the Keystone component is configured and running
+  In order to provide authentification and authorization service to  other Openstack components
+
+  Background:
+    Given project 'openstack' exists
+    And user 'test_user' exists
```

Comment:
So you are creating user in the Background? I understand it is something needed for all scenarios, but creation of user is something that we are testing, so shouldn't it be rather part of test scenario itself?